docs/design/kademlia_audit_gating.md

```diff
@@ -0,0 +1,116 @@
+# Kademlia Audit Gating
+
+## Abstract
+
+StorageNode B is added to StorageNode A's routing table only if StorageNode A trusts Satellite C and Satellite C has verified StorageNode B is not disqualified and has a high enough audit count and uptime count. If StorageNode B does not have this verification, it is added to StorageNode A's Routing Table Antechamber.
+
+## Background
+
+### White paper: 4.6.1 Mitigating Sybil attacks
+While we’ve adopted the proof-of-work scheme S/Kademlia proposes to partially address Sybil attacks, we extend Kademlia with an application specific integration to further defend our network. 
+
+
+Given two storage nodes, A and B, storage node B is not allowed to enter storage node A’s routing table until storage node B can present a signed message from a Satellite C that storage node A trusts claiming that B has passed enough audits that C trusts it (sections 4.13 and 4.15). This ensures that only nodes with verified disk space have the opportunity to participate in the routing layer.
```

Comment:
Kad supports multiple satellites and each satellite determines disqualification individually.  What happens when a SN is trusted by one satellite and not another?  Is it in the antechamber or the routing table?

Reply:
This is a good question. For now, I think it just needs to be trusted by one satellite. The vetting process is to mitigate sybil attacks, so if the node passed one satellite's vetting requirements: audit count, uptime count, not disqualified (there's also work for future parameters of minimum amount of data stored for a minimum amount of time), then it's probably a legitimate node

However, that brings up the question: what if there is a bad satellite? Maybe with the explicit purpose of setting an extremely low bar for issuing vouchers? Hopefully you wouldn't have that satellite in your trusted list and wouldn't accept a voucher from it

Reply:
Yeah, this would be a bit of a problem if you weren't whitelisting satellites... one faker satellite would invalidate the whole shebang.

Reply:
section 4.18 of the white paper vaguely discusses a satellite reputation system, so it might be the case that these faker satellites shouldn't make it into your trusted list in the first place

Reply:
Yeah right now we're relying on whitelisting satellites.